heat/core/statistics.py

```diff
@@ -1016,10 +1018,18 @@ def reduce_means_elementwise(output_shape_i: torch.Tensor) -> DNDarray:
 DNDarray.mean.__doc__ = mean.__doc__
 
 
-def median(x: DNDarray, axis: Optional[int] = None, keepdims: bool = False) -> DNDarray:
+def median(
+    x: DNDarray,
+    axis: Optional[int] = None,
+    keepdims: bool = False,
+    use_sketch_of_size: Optional[float] = None,
```

Comment:
Could we use two kwargs here for clarity? For example
```python
sketched: bool = False,
sketch_size: Optional[float] = 1.,
```

(same for percentile and preprocessing)

Reply:
taken into account, but with sketch_size = 1./MPI.COMM_WORLD.size as default as discussed this morning